src/Extensions/SiteTreeSubsites.php

```diff
@@ -117,10 +118,21 @@ public function updateCMSFields(FieldList $fields)
             $subsitesMap = new Map(ArrayList::create());
         }
 
+        // Check if we're viewing page history
+        // e.g. request to "/admin/historyviewer/schema/versionForm/2"
+        $viewingPageHistory = false;
+        if (class_exists(HistoryViewerController::class)) {
+            $urlSegment = HistoryViewerController::config()->get('url_segment');
+            $request = Controller::curr()->getRequest();
+            if (strpos($request->getURL(), "/$urlSegment/") !== false) {
+                $viewingPageHistory = true;
+            }
+        }
```

Comment:
```suggestion
        $viewingPageHistory = Controller::has_curr() && Controller::curr() instanceof HistoryViewerController;
```
I think this is a lot cleaner, and will catch unexpected edge cases (e.g. custom `HistoryViewerController` subclass with a different `url_segment`) or some custom route that uses `HistoryViewerController` without respecting `url_segment`.

Reply:
Ah yep much better

Have updated - note I got rid of the `Controller::has_curr()` as `var_dump(null instanceof stdClass);` is `false`

Reply:
For anyone seeing this in the future: We spoke offline about that - Steve has added `Controller::has_curr()` back in, since `Controller::curr()` throws an error if there's no current controller.